main.py

```diff
@@ -12,6 +12,95 @@
 
 # Import whatever libraries/modules you need
 
-import numpy as np
+import matplotlib.pyplot as plt
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import datasets, utils, layers, models
 
 # Your working code here
+
+# Load the CIFAR10 Dataset
+(trainX, trainY), (testX, testY) = datasets.cifar10.load_data()
+
+# Converting the labels into a new binary variable for the category
+trainY = utils.to_categorical(trainY)
+testY = utils.to_categorical(testY)
+
+# Convert from integer to float
+trainX = trainX.astype('float32')
+testX = testX.astype('float32')
+
+# Normalize the images to the range of 0 to 1
+trainX = trainX / 255.0
+testX = testX / 255.0
+
+# Constant Variables
+NUM_OF_CLASSES = 10
+HEIGHT = 32
+WIDTH = 32
+EPOCHS = 25
+BATCH_SIZE = 64
+
+# Next steps are initializing and developing the CNN model. The feature extraction part of the model is to be
+# created using a double convolutional layer, followed by a batch normalization layer and then a max pooling
+# layer. It will end with a dropout layer. This will be repeated 3 times with the first convolutional layer to
+# have 32 nodes, the second convolutional layer to have 64 nodes, and the third convolutional layer to have 128
+# nodes. The dropout layer will increase by 0.1 every time.
+
+model = models.Sequential()
+model.add(layers.Conv2D(32, (3, 3), activation='relu', input_shape=(HEIGHT, WIDTH, 3)))
```

Comment:
Please add comments explaining what each type of layer (conv, dropout etc.) Does